src/python/doc/cover_complex_sklearn_isk_ref.rst

```diff
@@ -0,0 +1,47 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+Scikit-learn class for cover complexes
+######################################
+
+.. list-table::
+   :widths: 40 30 30
+   :header-rows: 0
+
+   * - :Since: GUDHI 3.5.0
+     - :License: MIT
+     - :Requires: `Scikit-learn <installation.html#scikit-learn>`__
+
+We provide a scikit-learn class for computing cover complexes. Detailed examples on how to use this class in practice are available
+in the following `notebook <https://github.com/GUDHI/TDA-tutorial/blob/master/Tuto-GUDHI-cover-complex.ipynb>`_. 
+
+Example of Mapper cover complex computed from a point cloud
+-----------------------------------------------------------
+
+.. testcode::
+
+    import numpy as np
+    from sklearn.cluster import AgglomerativeClustering
+    from gudhi.sklearn import CoverComplex
+
+    X = np.array([[1,1],[1,1.5],[1,2],[1,2.5],[1,3],[1.5,2],[1.5,3],[2,2],[2,2.5],[2,3],[2,3.5],[2,4]])
+    F = np.array([[1,1,1,1,1,1.5,1.5,2,2,2,2,2],[1,1.5,2,2.5,3,2,3,2,2.5,3,3.5,4]]).T
+
+    Mapper = CoverComplex(complex_type="mapper", input_type="point cloud", cover="functional", colors=None, mask=0, filters=F, filter_bnds=np.array([[.5,2.5],[.5,4.5]]), resolutions=np.array([2,4]), gains=np.array([.3,.3]), clustering=AgglomerativeClustering(n_clusters=None, linkage='single', distance_threshold=.6))
+
+    _ = Mapper.fit(X)
```

Comment:
Couldn't it just be `Mapper.fit(X)` (without the `_ =` ?).

Reply:
I was using `_ = ...` so that the code does not print the output, but maybe there is a more elegant way for doing this.

Reply:
I thought it only automatically printed the output of the last line of a cell when using an interactive version of python. Was it printing something in the doc without it?